governance/program/Cargo.toml

```diff
@@ -14,16 +14,23 @@ test-bpf = []
 
 
 [dependencies]
```

Comment:
nit: keeping the dependencies sorted alphabetically helps (me) with readability, ever so slightly

Reply:
Done